src/datasources/alarms/types/types.ts

```diff
@@ -4,8 +4,13 @@ export interface AlarmsQuery extends DataQuery {
   queryType?: QueryType;
 }
 
+export interface AlarmsVariableQuery extends AlarmsQuery {
+  queryBy?: string;
+}
+
 export enum QueryType {
   AlarmsCount = 'AlarmsCount',
+  ListAlarms = 'ListAlarms'
```

Comment:
```suggestion
  ListAlarms = 'List Alarms'
```

There is an [active PR](https://github.com/ni/systemlink-grafana-plugins/pull/412/files#diff-02c3364825d2451bb50b9bfa792805099f4ac7d873cbe67a3b298070054edc1f:~:text=AlarmsCount%20%3D%20%27Alarms%20Count%27%2C) in progress to update the AlarmsCount value.